src/container/srv_csum_recalc.c

```diff
@@ -183,9 +184,15 @@ csum_agg_verify(struct csum_recalc *recalc, struct dcs_csum_info *new_csum,
 	 * starting at the correct offset of the checksum array for the input
 	 * segment.
 	 */
-	return !memcmp(new_csum->cs_csum,
-		       &recalc->cr_phy_csum->cs_csum[j * new_csum->cs_len],
-		       new_csum->cs_nr * new_csum->cs_len);
+	match = !memcmp(new_csum->cs_csum,
```

Comment:
minor nit.   I prefer to use == 0 since memcmp isn't a boolean.  Either works though.